lib/takeaway_shop.rb

```diff
@@ -0,0 +1,41 @@
+require 'csv'
+require_relative 'menu'
+
+class Shop
+
+	attr_reader :menu
+
+	def initialize(menu_items = Menu.new.upload)
+		@menu_items = menu_items
+		@menu = ""
+	end
+
+	def view_menu
+		menu_header
+		print_menu
+		menu_footer
+	end
+
+
+private
+
+	def index_items
+		@menu_items.map.with_index(1) do |menu_item, i|
+			@menu << "#{i}		#{menu_item[:item]}		#{menu_item[:price]}			#{menu_item[:description]}\n"
+		end
+	end
+
+	def print_menu
```

Comment:
You have 3 methods here that all include `menu` in the name.  Isn't that a clue that they belong in `Menu`?